marshmallow/validate.py

```diff
@@ -271,6 +274,48 @@ def __call__(self, value):
         return value
 
 
+class OneOf(object):
+    """Validator which succeeds if ``value`` is a member of ``choices``.
+
+    :param iterable choices:
+        A sequence of valid values.
+    :param iterable labels:
+        Optional sequence of labels to pair with the choices.
+    :param str error:
+        Error message to raise in case of a validation error.
+    """
+    def __init__(self, choices, labels=None, error=None):
+        self.choices = choices
+        self.labels = labels if labels is not None else []
+        self.error = error or 'Not a valid choice.'
```

Comment:
Perhaps we can have a more descriptive error message that can be templated (like in `Range` and `Length`). Something like: `'{val} is not one of {choices}'`